Cubical/ZCohomology/Groups/Wedge.agda

```diff
@@ -45,11 +41,20 @@ module _ {ℓ ℓ'} (A : Pointed ℓ) (B : Pointed ℓ') where
     bijIso : BijectionIso (coHomGr 1 (A ⋁ B)) (×coHomGr 1 (typ A) (typ B))
     BijectionIso.map' bijIso = I.i 1
     BijectionIso.inj bijIso =
-       sElim (λ _ → isOfHLevelΠ 2 λ _ → isOfHLevelPath 2 setTruncIsSet _ _)
-              λ f inker → helper ∣ f ∣₂ (I.Ker-i⊂Im-d 0 ∣ f ∣₂ inker)
+      sElim (λ _ → isSetΠ λ _ → isProp→isSet (setTruncIsSet _ _))
+            λ f inker → helper ∣ f ∣₂ (I.Ker-i⊂Im-d 0 ∣ f ∣₂ inker)
     BijectionIso.surj bijIso =
-      sigmaElim (λ _ → isOfHLevelSuc 1 propTruncIsProp)
-                 λ f g → I.Ker-Δ⊂Im-i 1 (∣ f ∣₂ , g) (isOfHLevelSuc 0 (isContrHⁿ-Unit 0) _ _)
+      prodElim (λ _ → isProp→isSet propTruncIsProp)
+                λ f g → ∣ ∣ helpFun f g {!!} ∣₂ , refl ∣₁
```

Comment:
@aljungstrom I tried to inline the proof and simplify it, but I can't figure out what to do with this hole...

Reply:
I'll have a look:-)

Reply:
You can skip the elimination. That was caused by some strange issue which seems to have been magically resolved. Try:
BijectionIso.surj bijIso p = I.Ker-Δ⊂Im-i 1 p (isOfHLevelSuc 0 (isContrHⁿ-Unit 0) _ _)

Reply:
Actually I think there are many places where one could remove these unnecessary elimminations. I have no idea why it terminates now and not before.

Reply:
Might have to do with me being more careful about rec/elim... Or any of the other many optimizations we've found :-)